server/server.js

```diff
@@ -114,6 +114,18 @@ module.exports = port => {
         res.sendStatus(200);
     });
 
+    /* Check if it's my turn */
+    app.get('/api/myTurn', (req,res) => {
+        var user = currentUsers.find((user) => user.username === req.session.user);
```

Comment:
One comment I would make is to try to avoid using `var` wherever possible. I would also favour `const` (or `let` if it needs to be mutable).